src/index.ts

```diff
@@ -70,26 +81,86 @@ const main = async () =>
 
   info(`Executed installer. (Exit code ${exitCode})`);
 
+  const wasmerDir = join(homedir(), ".wasmer");
+
+  if (isWindows) {
+    startGroup("Download and Add WAPM-Cli to the PATH.");
+
+    //Get the wapm version
+    const wapmVersion = getInput("wapm_version");
+
+    const tmpWapm = path({
+      suffix: ".tar.gz",
+    });
+
+    //Download the latest installer
+    const progressHandler = (data: {
+      transferred: number;
+      total: number;
+      percent: number;
+    }) => {
+      //Round
+      const transferred = Math.round(data.transferred / 1000);
+      const total = Math.round(data.total / 1000);
+      const percent = Math.round(data.percent * 100);
+
+      //Log
+      info(`Downloaded ${transferred}K out of ${total}K (${percent}%)`);
+    };
+
+    await pipeline(
+      got
+        .stream(
+          `https://github.com/wasmerio/wapm-cli/releases/download/v0.5.9/wapm-cli-windows-amd64.tar.gz`
+        )
+        .on("downloadProgress", progressHandler),
+      createWriteStream(tmpWapm, {
+        mode: 0o655,
+      })
+    );
+    const currentDir = process.cwd();
+    const wasmerBinDir = join(homedir(), ".wasmer", "bin");
+    const wapmExecutableDir = join(currentDir, "bin");
+
+    tar
+      .x({
+        file: tmpWapm,
+        cwd: currentDir,
+      })
+      .then((file) => {
+        console.log(file);
+        fs.copyFile(
+          join(wapmExecutableDir, "wapm.exe"),
+          join(wasmerBinDir, "wapm.exe"),
+          (err) => {
+            if (err) throw err;
+            console.log("WAPM-Cli was copied to Wasmer bin directory.");
+          }
+        );
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+    endGroup();
+  }
```

Comment:
This is now automatically handled by the windows installer (we merged https://github.com/wasmerio/wasmer-install/pull/15 which addressed that). So we don't need this changes any longer

Reply:
Everything else just lint changes, so if we don't need this - I think its safe to close this PR @dynamite-bud